source/common/access_log/access_log_manager_impl.h

```diff
@@ -34,9 +35,9 @@ class AccessLogManagerImpl : public AccessLogManager {
                        Event::Dispatcher& dispatcher, Thread::BasicLockable& lock,
                        Stats::Store& stats_store)
       : file_flush_interval_msec_(file_flush_interval_msec), api_(api), dispatcher_(dispatcher),
-        lock_(lock), file_stats_{ACCESS_LOG_FILE_STATS(
-                         POOL_COUNTER_PREFIX(stats_store, "access_log_file."),
-                         POOL_GAUGE_PREFIX(stats_store, "access_log_file."))} {}
+        lock_(lock), file_stats_{
+                         ACCESS_LOG_FILE_STATS(POOL_COUNTER_PREFIX(stats_store, "filesystem."),
```

Comment:
I can't remember what type of deprecation we have in place for changing stats namespaces. Given that we are reviving a namespace that existed before we might be inclined to be ok with this. However, I can imagine that people that run off of master might be taken off guard by this stat name changing?

Reply:
This stat was changed from this name to the current name without deprecation (because no one noticed the name got changes in a relatively big structural change). This was briefly discussed on Slack and it seemed ok to just change it back since no one noticed the first time.